int32 particle_id
bool is_dynamic
---
bool success